src/style.css

```
@@ -0,0 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  min-width: 300px;
+  height: 300px;
+  font-size: 100px;
+  color: #fff;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+}
+
+.block--1 {
+  background-color: hsl(0, 100%, 50%);
+}
+
+.block--2 {
+  background-color: hsl(0, 100%, 40%);
+}
+
+.block--3 {
+  background-color: hsl(0, 100%, 30%);
+}
+
+.block--4 {
+  background-color: hsl(0, 100%, 20%);
+}
+
+.block--5 {
+  background-color: hsl(0, 100%, 10%);
+}
+
+.block--6 {
+  background-color: hsl(0, 100%, 0%);
+}
+
+@media screen and (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media screen and (min-width: 600px) {
```
Нижній медіа запит перекриє цей